src/model/fitmodel.py

```diff
@@ -158,16 +158,16 @@ class FitModel:
 
     all_models = zip(list_of_models_names, list_algorithms, list_feature_imp_cols)
 
-    def __init__(self, all_models, df_encode: pd.DataFrame, list_of_models: list = None, data_path_parent_level:
-    int = 1):
+    def __init__(self, df_encode: pd.DataFrame, list_of_models: list = None, data_path_parent_level:
```

Comment:
For the type hint, can you use List[str]? it is important to denote what is inside the list.

Reply:
this is a good point, updated!